Content.Shared/Actions/ActionTypes/ActionPrototypes.cs

```diff
@@ -0,0 +1,56 @@
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Actions.ActionTypes;
+
+// These are just prototype definitions for actions. Allows actions to be defined once in yaml and re-used elsewhere.
+// Note that you still need to create a new instance of each action to properly track the state (cooldown, toggled,
+// enabled, etc). The prototypes should not be modified directly.
+//
+// If ever action states data is separated from the rest of the data, this might not be required
+// anymore.
+
+[Prototype("worldTargetAction")]
+public sealed class WorldTargetActionPrototype : WorldTargetAction, IPrototype
+{
+    [IdDataFieldAttribute]
+    public string ID { get; } = default!;
+
+    // This is a shitty hack to get around the fact that action-prototypes should not in general be sever-exclusive
+    // prototypes, but some actions may need to use server-exclusive events, and there is no way to specify on a
+    // per-prototype basis whether the client should ignore it when validating yaml.
+    [DataField("serverEvent", serverOnly: true)]
+    public WorldTargetActionEvent? SeverEvent
```

Comment:
server

Reply:
Fixed in [`fb22b64` (#7508)](https://github.com/space-wizards/space-station-14/pull/7508/commits/fb22b648964b4f7e305a1306f835d880c7459ab6)